spec/factories.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :user do
+    email "user@example.com"
+    password "123456"
+  end
+
+  factory :book do
+    title "sdf"
```

Comment:
Pls use real titles and text in factories. Or you can add gem 'faker', it can generate almost real strings